Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -53,3 +53,20 @@ typedef NS_ENUM(NSUInteger, OCTMessageFileType) {
      */
     OCTMessageFileTypeReady,
 };
+
+typedef NS_ENUM(NSInteger, OCTMessageCallType) {
```

Comment:
I had to make this into an `NSInteger` instead of `NSUInteger`. I'm not sure how it worked for `OCTMessageFileType` but not for this one since I kept getting 
`failed: caught "RLMException", "Can't persist property 'type' with incompatible type. Add to ignoredPropertyNames: method to ignore."`